lib/bowling_score.rb

```diff
@@ -0,0 +1,40 @@
+class Bowling
+
+  def initialize
+    @score = []
+    @frame_no = 1 # game ends when frame_no = 11
+    @strike = false
+    @spare = false
+  end
+
+  def frame(roll_1, roll_2) # roll_1 and roll_2 are integers 
+                            # entered by player.
+    if @strike == true
+      roll_1 = (roll_1 * 2)
+      roll_2 = (roll_2 * 2)
+      @strike = false
+    end
```

Comment:
Brilliant method to calculate strike bonus, much simpler than using indices. Be good to test for potential bonus round.